Resources/Prototypes/Entities/Objects/Devices/station_beacon.yml

```
@@ -1,11 +1,11 @@
 - type: entity
   parent: BaseItem
-  id: DefaultStationBeacon
+  id: BaseStationBeacon 
   name: station beacon
   description: A small device that transmits information to station maps. Can be configured.
   placement:
     mode: SnapgridCenter
-  suffix: General
+  suffix: Not teleport point, Ninja bombing target blacklist
```

This just seems more clear to me, but I am open to other suggestions

```suggestion
  suffix: Not teleport point, Not ninja bombing target 
```